README.md

```diff
@@ -1,436 +1,95 @@
-## Metrilo custom integration tutorial ##
 
-_This guide is from developers for developers. If your are not a developer, please find one and come back later :)_
 
+# Metrilo custom integration tutorial
 
+**This guide is from developers for developers. If your are not a developer, please find one and come back later :)**
 
-**:warning: :warning: :warning:
-We strongly recommend doing this on Staging/Beta environment first to make sure the integration is smooth and nothing breaks down.**
+:warning: **We strongly recommend doing this on Staging/Beta environment first to make sure the integration is smooth and nothing breaks down.**
 
-Metrilo provides a javascript library for integration with your custom e-commerce solution.
-The same library is used for our Magento and Woocommerce plugins so they can be a valuable source when integrating Metrilo with your website. You can find them on:
-- [Magento Plugin](https://github.com/Metrilo/Magento-Plugin)
-- [Woocommerce Plugin](https://github.com/Metrilo/WooCommerce-Plugin)
 
-This guide consists of 2 parts:
+To do your custom integration with Metrilo and send data to our servers you have to use the **Metrilo API**. There is a documentation about it written according to the [OpenAPI Specification](https://swagger.io/docs/specification/about/). Please copy the content of this [file](metrilo_open_api_specification.yml) to the [swagger editor](https://editor.swagger.io/).
 
-- [Front-end Metrilo integration](#front-end-metrilo-integration)
-- [Back-end Metrilo integration](#back-end-metrilo-integration)
+There are 3 things you have to do to make Metrilo work on your website. These are:
+* [Import of resources](#import)
+* [Frontend integration](#frontend-integration)
+* [Backend integration](#backend-integration)
 
+## Import
```

Comment:
The import docs should be moved to a separate file and you should add code examples.

Reply:
It is not that much info to make everything in a separate file. Also, the documentation(swagger) is enough explanatory in my opinion. What should I do as example? How to make http request with what is describe in swagger? Moreover, what about the language?